client/src/components/EntryInput.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+const JournalInput = styled.input`
+  all: unset;
+  height: 22px;
+  width: 280px;
+  display: flex;
+  height: 18px;
+  padding: 2px;
+  background-color: ${props => props.theme.colors.secondary};
+  border-radius: 6px;
+  align-self: center;
+  font-size: 14px;
+  border: none;
+  outline: none;
+  opacity: 0.75;
+`;
+function EntryInput(props) {
+  return (
+    <div {...props}>
```

Comment:
Why do you have a `div` here? I think you can directly export the styled.input if you change the display to `block`

Reply:
thank you...